numba/np/numpy_support.py

```diff
@@ -391,25 +391,46 @@ def set_output_dt_units(inputs, outputs):
         leads to an output of timedelta output. However, for those that do,
         the unit of output must be inferred based on the units of the inputs.
 
-        At the moment this function takes care of the case where all
-        inputs have the same unit, and therefore the output unit has the same.
-        If in the future this should be extended to a case with mixed units,
-        the rules should be implemented in `npdatetime_helpers` and called
-        from this function to set the correct output unit.
+        At the moment this function takes care of two cases
+         a) where all inputs are timedelta with the same unit (mm), and
+         therefore the output has the same unit.
+         If in the future this should be extended to a case with mixed units,
```

Comment:
The `If` in this phrase is correct? I think it should be `In the future this should be extended ...`?

Reply:
thanks, you are right, it was wrong. I kept the if, but changed the should.